python/cuml/tests/test_pca_sign_flip.py

```diff
@@ -0,0 +1,209 @@
+import numpy as np
+import pandas as pd
+
+from sklearn.datasets import load_breast_cancer
+from sklearn.model_selection import train_test_split
+from sklearn.decomposition import PCA as skPCA
+from cuml.decomposition import PCA as cuPCA
+
+def test_pca_sign_flip():
+    X, y = load_breast_cancer(return_X_y=True, as_frame=True)
+    # X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=13)
+    sk_pca = skPCA(n_components=10, random_state=13)
+    sk_pca_res = sk_pca.fit_transform(X)
+    print(sk_pca_res)
```

Comment:
Remove the prints and the comments.
The test should be enough by itself to determine success/failure